index.js

```diff
@@ -1,7 +1,56 @@
 // Iteration 1: Names and Input
+console.log("I`m ready");
 
+let hacker1 = "The driver's name is Leal";
```

Comment:
Here, instead of passing the variable as the whole string, you could've given just your name and console.log the string after, like:

```
const hacker1 = 'Leal';
 console.log(`The driver's name is ${hacker1}.`);
 const hacker2 = 'Duarte';
 console.log(`The navigator's name is ${hacker2}.`)
```